app.rb

```diff
@@ -0,0 +1,143 @@
+require './book'
+require './student'
+require './teacher'
+require './person'
+require './rental'
+
+class App
+  include Inputs
```

Comment:
- [ ] There is no function called `Inputs`, kindly recheck what you are including in the `App` class. 👍